### PR TITLE
Issue 13895 - Declaration grammar is insufficient

### PR DESCRIPTION
**Pull request by 9rnsr**
https://issues.dlang.org/show_bug.cgi?id=13895

### CODE REVIEW

File: statement.dd [hunk 1]
```diff
@@ -206,7 +206,7 @@ $(H3 $(LEGACY_LNAME2 DeclarationStatement, declaration-statement, Declaration St
 
 $(GRAMMAR
 $(GNAME DeclarationStatement):
-    $(GLINK2 declaration, Declaration)
+    $(GLINK2 declaration, StorageClasses)$(OPT) $(GLINK2 declaration, Declaration)
```
Comment (WalterBright) [inline statement.dd]: StorageClasses should not link to #declaration. Should be: `$(GLINK StorageClasses)`

Comment (9rnsr) [inline statement.dd]: This is in `statement.dd`.

**Comment by 9rnsr** [comment]
Sorry I had forgotten to change `grammer.dd` at the same time. Fixed.

**Comment by andralex** [comment]
Thanks, Kenji-san!

**Pull request merged**